src/greptimeai/extractor/openai_extractor/__init__.py

```diff
@@ -80,7 +80,7 @@ def get_user_id(**kwargs) -> Optional[str]:
 
     @staticmethod
     def extract_usage(
-        model: Optional[str], usage: Optional[Dict[str, int]]
+        model: Optional[str], usage: Any
```

Comment:
`usage: Any` may make static type checking weak to find potential error. Is it ok to keep the original `Optional[Dict[str, int]]`?